From 093c88817397425ee4c2333c469467229a46c9e1 Mon Sep 17 00:00:00 2001
From: Yuan Chao <yuanc.fnst@cn.fujitsu.com>
Date: Mon, 5 Aug 2019 01:00:58 +0900
Subject: [PATCH 7/7] python: Add variables to tests

* configure.ac, lang/python/Makefile.am:
  New variable to `lang/python', set to `lang/python' if RUN_LANG_PYTHON_TESTS

Upstream-Status: Submitted [gnupg-devel@gnupg.org]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Signed-off-by: Yuan Chao <yuanc.fnst@cn.fujitsu.com>
---
 configure.ac            | 5 +++++
 lang/python/Makefile.am | 3 +++
 2 files changed, 8 insertions(+)

diff --git a/configure.ac b/configure.ac
index bd85886..b5ae42e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -533,6 +533,11 @@ AC_ARG_ENABLE(g13-test,
          run_g13_test=$enableval)
 AM_CONDITIONAL(RUN_G13_TESTS, test "$run_g13_test" = "yes")
 
+run_lang_python_test="yes"
+AC_ARG_ENABLE(lang-python-test,
+  AC_HELP_STRING([--disable-lang-python-test], [disable Python regression test]),
+         run_lang_python_test=$enableval)
+AM_CONDITIONAL(RUN_LANG_PYTHON_TESTS, test "$run_lang_python_test" = "yes")
 
 # Checks for header files.
 AC_CHECK_HEADERS_ONCE([locale.h sys/select.h sys/uio.h argp.h stdint.h
diff --git a/lang/python/Makefile.am b/lang/python/Makefile.am
index 551deee..0fd555e 100644
--- a/lang/python/Makefile.am
+++ b/lang/python/Makefile.am
@@ -23,7 +23,10 @@ EXTRA_DIST = \
 	gpgme.i \
 	helpers.c helpers.h private.h
 
+SUBDIRS = . examples doc src
+if RUN_LANG_PYTHON_TESTS
 SUBDIRS = . tests examples doc src
+endif
 
 .PHONY: prepare
 prepare: copystamp
-- 
2.7.4

